honeycombio/example-greeting-service
# feat: Dockerize dotnet services

## Which problem is this PR solving?
This PR dockerizes the .NET services and changes the Tiltfile to run .NET services through Docker.

## Short description of the changes
- Update docker-compose.yml for .NET services to be able to run on its own
- Updated Tiltfile to run .NET services with the docker compose

## How to test
1. Pull the changes locally
2. Run `tilt up dotnet` from the root directory
3. Check that the greeting service works

Things to test:
- [x] `tilt up dotnet` works from the root directory
- [x] `docker-compose up` works from the `dotnet` directory
- [x] Other tilt commands work for services in other languages (e.g. `tilt up go`)
